app/views/handlers/block.py

```diff
@@ -114,3 +109,10 @@ def _update_section_completeness(self, location: Optional[Location] = None):
             section_id=location.section_id,
             list_item_id=location.list_item_id,
         )
+
+    def _set_started_at_metadata(self):
+        collection_metadata = self._questionnaire_store.collection_metadata
+        if not collection_metadata.get("started_at"):
+            started_at = datetime.utcnow().isoformat()
+            logger.info("Survey started", started_at=started_at)
+            collection_metadata["started_at"] = started_at
```

Comment:
Mutating the metadata in this way doesn't seem great. Can't started_at be set when the collection_metadata is first created?

Reply:
This isn't metadata, this is collection metadata that was created for this purpose. It's done this way so that it is only set once they have POSTed a page. This also isn't new, it was previously called from the route.